### ISSUE TITLE
[$20][Safari] [voiceover] Error message in Support window field are not read out by voiceover

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3082]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing 
2. click  support button 
3. hit tab key 
4. Hit enter
5. Hit tab key 
6. observe

**Expected Result**
Error message should be read out to notify where the error has occurred

**Actual Result**
Visible Error message in text box field are not read out by voiceover

**Device**: Desktop/Labtop
**Browser**: Safari

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.3.1 Error Identification
**Screen Reader Used**: Voiceover
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=09f22811%2D138e%2D4d3b%2Da8d5%2Dd3693a709128&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2FCreeya%2Fbug%2D15

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3082 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3082 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[504]: Failed to create challenge.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup. PR against `qa-accessibility` branch.

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3082 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
[500]: Challenge for the updated issue 3082 is creating, rescheduling this event<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097876 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097876 has been updated - it has been assigned to mfikria.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by tc-mfikria** [comment]
@kkartunov It cannot be fixed because the form is a zendesk form that is not part of `connect-app`

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
